views/transaccion_completa_mall_deferred/form.pug

```diff
@@ -9,25 +9,25 @@ block step
         | se desea generar una transacción.
     .col.m7
       .card
-        form#card(action="/transaccion_completa_mall_deferred/create", method="POST")
```

Comment:
is correct change to _transaccion_completa_, if this is _transaccion_completa_mall_deferred_ form?